From bbdadb803b19f002e76e7bb5ce3faf770dd7c413 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@cn.fujitsu.com>
Date: Wed, 19 Aug 2015 12:33:41 +0900
Subject: [PATCH] don't swallow errors

Further fixup to the configure scripts to not swallow errors

Upstream-Status: Pending
---
 src/configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/configure.in b/src/configure.in
index 82d64d6..0b23df5 100644
--- a/src/configure.in
+++ b/src/configure.in
@@ -219,7 +219,7 @@ main(void)
 	return 2;
     }
 }], rpppoe_cv_pack_bitfields=normal, rpppoe_cv_pack_bitfields=rev,
-$ECHO "no defaults for cross-compiling"; exit 0)
+$ECHO "no defaults for cross-compiling"; exit 1)
 fi
 ])
 
-- 
1.8.4.2

